src/ui_ng/lib/src/helm-chart/versions/helm-chart-version.component.ts

```diff
@@ -161,7 +158,11 @@ export class ChartVersionComponent implements OnInit {
   deleteVersions(versions: HelmChartVersion[]) {
     if (versions && versions.length < 1) { return; }
     let versionObs = versions.map(v => this.deleteVersion(v));
-    Observable.forkJoin(versionObs).finally(() => this.refresh()).subscribe();
+    Observable.forkJoin(versionObs).finally(() => this.refresh()).subscribe(res => {
+      if (this.chartVersions.length === versionObs.length) {
```

Comment:
Is this can be the condition of delete all the versions?

Reply:
In this scenario, yes